src/web/components/SharingPermission/SharingPermissionsTable.scss

```diff
@@ -6,12 +6,12 @@
     justify-content: space-between;
     align-items: center;
     margin-bottom: 16px;
-  }
 
-  .sharing-permissions-table h1 {
-    font-weight: 600;
-    font-size: 1.125rem;
-    line-height: 1.375rem;
+    h1 {
```

Comment:
As discussed - make the element a h2 instead and update the size of h2 across the site (it's only used in one other place which should also be 1.125rem).